### PR TITLE
Make deployer-model configurable

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/jenkins/ardana/ansible/templates/input-model-servers.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
     - id: ccn-0001
       ip-addr:  {{ deployer_mgmt_ip }}
-      role: LITE-CONTROLLER-ROLE
+      role: CONTROLLER-ROLE
```
Comment (rhafer) [inline scripts/jenkins/ardana/ansible/templates/input-model-servers.yml]: I guess this requires a change to the deployerincloud-lite in "ardana-input-models"? Is there a patch for that already?

Comment (dirkmueller) [reply scripts/jenkins/ardana/ansible/templates/input-model-servers.yml]: hmm, you mean not having different role names between the -lite and non-lite model? I think that would make sense.

Comment (rhafer) [reply scripts/jenkins/ardana/ansible/templates/input-model-servers.yml]: Well, if you just change it here things will fail for the -lite model, as the roles are referenced in a few other files of the model.

And IMO just aligning them for the deployerincloud-* models makes sense.

Comment (dirkmueller) [reply scripts/jenkins/ardana/ansible/templates/input-model-servers.yml]: https://gerrit.suse.provo.cloud/2862

Comment (nicolasbock) [reply scripts/jenkins/ardana/ansible/templates/input-model-servers.yml]: Does this naming effect other models or are we considering here to only ever deploy the -lite and the -non-lite models?

Comment (dirkmueller) [reply scripts/jenkins/ardana/ansible/templates/input-model-servers.yml]: @nicolasbock we need to have that discussion with someone that has knowledge on the design of input models

Comment (nicolasbock) [reply scripts/jenkins/ardana/ansible/templates/input-model-servers.yml]: Ok, sounds good.

File: scripts/jenkins/ardana/ansible/repositories.yml [hunk 2]
```diff
@@ -20,9 +20,9 @@
       name: SLES12SP3-Updates
 
   # Devel:Cloud:8
-  - name: Add Devel:Cloud:8:Staging media
+  - name: "Add {{ cloudsource }} media"
```
Comment (toabctl) [inline scripts/jenkins/ardana/ansible/repositories.yml]: Can you add a default value for this new variable in this file? That way, repository.yml can be used without the ```hosts``` file (which is generated by the jenkins job)

File: scripts/jenkins/ardana/ansible/init.yml [hunk 3]
```diff
@@ -85,7 +85,7 @@
 
   - name: Update servers in Model
     template:
-      src: input-model-servers.yml
+      src: "{{ deployer_model }}/input-model-servers.yml"
```
Comment (toabctl) [inline scripts/jenkins/ardana/ansible/init.yml]: Please add a default for this variable (similar to the other comment)

File: jenkins/ci.suse.de/ardana-job.yaml [hunk 4]
```diff
@@ -68,7 +74,7 @@
           pushd ansible
           cat << EOF > hosts
           [hosts]
-          $DEPLOYER_IP ansible_user=root
+          $DEPLOYER_IP ansible_user=root -e deployer_model=${model} -e cloudsource=${cloudsource}
```
Comment (toabctl) [inline jenkins/ci.suse.de/ardana-job.yaml]: that looks wrong.

File: jenkins/ci.suse.de/ardana-job.yaml [hunk 5]
```diff
@@ -44,6 +44,12 @@
           description: >-
             This name will become the build name of the job.
 
+      - string:
+          name: cloudsource
+          default: 'SUSE-OpenStack-Cloud-8-devel-staging'
+          description: >-
+            This is used as input repository (from provo-clouddata) for testing
+
```
Comment (toabctl) [inline jenkins/ci.suse.de/ardana-job.yaml]: Isn't ```deployer_model``` missing as parameter here?

Comment (rhafer) [reply jenkins/ci.suse.de/ardana-job.yaml]: There is a parameter called "model" on line 35 which was added as part of 88525d5a77

**Pull request merged**